### ISSUE TITLE
Register Online and Registration Form need space for Accessibility Requests

### ISSUE THREAD

**Issue by Venar** [AnimeTwinCities/registration#33]
# Definition of Done
## Registration Form (Online)
The online form should have a checkbox to indicate that the attendee needs accessibility assistance. It should then have a multiple choice list of options for them to specify how we can assist them. With an other options for them to type out

## Registration Form (PDF)
The PDF Form should have a check box indicating they require accessibility needs and then have a space for them to write out how we can assist them.
